app/models/invenio_rdm_record_converter.rb

```diff
@@ -263,4 +258,11 @@ def rights(license_urls)
       }
     end
   end
+
+  def version(content)
+    return "" unless content.has_versions?
+    version_number = content.versions.all.length
+
+    "v#{version_number}.0.0"
```

Comment:
Simply replicating the documentation's example? When would a GenericFile's versions be more than one? Just curious, and we can omit testing this scenario.